### PR TITLE
feat(man): variadic args, fish pages, correct path handling

### PR DESCRIPTION
**Pull request by SeparateRecords**
Also fixes slightly incorrect manpath handling, and if your login shell is fish then it shows fish-specific completions.

Uses possibly the longest shell script we've added in autocomplete to date! (this is the evaluated version, the raw version is even worse)

```bash
while IFS=':' read -ra manpath; do
  for path in "${manpath[@]}"; do
    ls -1 "$path/man"[18] 2>/dev/null
  done
done <<< "$(man -w)" | sort -u
```

image

### CODE REVIEW

**Comment by SeparateRecords** [comment]
Will take another look at this and see if there's a simpler way. Surely... surely there is, right?

**Comment by fedeci** [comment]
@SeparateRecords We can add this one for the moment! I had a pending review I forgot to publish, sorry!

**Comment by SeparateRecords** [comment]
> @SeparateRecords We can add this one for the moment! I had a pending review I forgot to publish, sorry!

That's all good! It fixed a bug that was annoying me but I'm not 100% happy with the implementation, will take another stab at it later

File: src/man.ts [hunk 1]
```diff
@@ -13,49 +15,66 @@ const sectionIcon = "📑";
...
-        return !(line.length == 0 || line.startsWith("/"));
-      })
-      .map((line) => {
-        return {
-          name: line,
-          description: "Manual page",
-          icon: "fig://icon?type=string",
-        };
+    const out = await executeShellCommand(
+      `while IFS=':' read -ra manpath; do for path in "\${manpath[@]}"; do \\ls -1 "$path/man"${sectionGlob} 2>/dev/null; done; done <<< "$(${manpathCommand})" | sort -u`.trim()
```
[the remark this answers is not in the document]
Comment (SeparateRecords) [reply src/man.ts:39]: Honestly I didn't even realise that was there, hold-over from when it was a multiline script. Vim mode so I just hit shift+J to join the lines 😄

**Pull request closed without merging**